props/saber_BC_buttons.h

```diff
@@ -753,23 +762,58 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     return true;
 
 // Blaster Deflection
+    // original sa22c mode if ENABLE_AUTO_SWING_BLAST is not defined
```

Comment:
I think you need to sync before I merge, as this seems to undo some changes I just merged.

Reply:
Ok, i think this is right now. Made the changes here too so it doesn't conflict?
Although it looks like I hit GO twice :/